### PR TITLE
GitHub issue and PR templates

### PR DESCRIPTION
**Pull request by Ticki**
**Problem**: The PRs tend to be very little descriptive and non-explaining without having to look at the diffs.

**Solution**: Introduce templates to encourage rich information about the changes, to increase quality of the issues and PR descriptions.

**Changes introduced by this pull request**:

- Add a PR template. See [Github's blog post](https://github.com/blog/2111-issue-and-pull-request-templates). This template is to encourage non-small PRs to contain extra useful information about the changes.

- Add an issue template, encouraging users to describe the reproduction steps and the expected behavior.

**Drawbacks**: This might reduce the number of micro PRs (~3 lines), since describing the changes is not a programmers favorite task. In order to combat this, we add a small message in the bottom stating that the description is unnecessary for smaller PRs.

**TODOs**: Possibly improvement to the fields.

**Fixes**: This does not fix any specific issues.

**State**: Ready.

**Blocking/related**: This is a part of the documentation effort going on. Related to #525.

### CODE REVIEW

**Comment by Ticki** [comment]
Things to discuss:

Should there be a `related` section? Should there be a `discussion` section?

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,13 @@
+**Reproduction**: [describe how you are able to reproduce ("trigger") this bug/issue.]
+
+**Expected behavior**: [describe the behavior you would expect the repro to yield.]
+
+**Actual behavior**: [describe the actual behavior, which is presented through the repro.].
+
+**Build information**: [output of `rustc -V`, `git rev-parse HEAD`, `qemu-i386 -version`, `uname -a`, etc.]
+
+**Misc**: [...]
```
Comment (stratact) [inline .github/ISSUE_TEMPLATE]: [optional: for other relevant information that should be known or cannot be describe in the other fields]

**Comment by nounoursheureux** [comment]
Do you think the other projects (ion, orbclient, orbtk,...) should use the templates too ?

**Comment by Ticki** [comment]
Feel free to steal it.

**Pull request merged**